tests/clickhouse-test

```diff
@@ -2353,7 +2408,13 @@ class BuildFlags:
     POLYMORPHIC_PARTS = "polymorphic-parts"
 
 
+# Release and non-sanitizer build
+RELEASE_BUILD = False
```

Comment:
I think naming the variable something more descriptive such as `RELEASE_NON_SANITIZED` or something along those lines helps understanding. The only implications of this var is the time we're sleeping. We could have at some point released but sanitized builds where you'd still want to sleep longer, right?

Reply:
>I think naming the variable something more descriptive such as RELEASE_NON_SANITIZED or something along those lines helps understanding

OK, I don't have strong filling about it - will use your suggestiong

> We could have at some point released but sanitized builds where you'd still want to sleep longer, right?

I don't think so and I hope that there will be no more extra switches for sleeping, this will only make it less readable then

Reply:
No worries. I thought at first if it was called just `RELEASE` it'd rather be something along these lines, for consistency:

```python
RELEASE_BUILD = BuildFlags.RELEASE in result
```

But then when I realized the sleep amount it was probably due to either debug or because of sanitizers, I suggested the name change instead. It's not a major thing, in any case